backend/app/views/spree/admin/orders/customer_details/edit.html.erb

```diff
@@ -1,11 +1,10 @@
 <%= render :partial => 'spree/admin/shared/order_tabs', :locals => { :current => 'Customer Details' } %>
 
 <% content_for :page_title do %>
-  <i class="fa fa-arrow-right"></i> <%= Spree.t(:customer_details) %>
+  &#x2f; <%= Spree.t(:customer_details) %>
 <% end %>
 
 <% content_for :page_actions do %>
```

Comment:
Ah, you're absolutely right. We check with `content_for? :page_actions` and only render the div if something is present. We could remove the check or advice people to add `content_for :page_actions` to their overrides. I would vote for to go with latter. It was never guaranteed that this div exists anyway. Thoughts @jhawthorn ?